rpm/libcap-ng.spec

```diff
@@ -94,22 +97,22 @@ make check
 %files
 %license COPYING.LIB
 %{_libdir}/libcap-ng.so.*
-#%%{_libdir}/libdrop_ambient.so.*
-%attr(0644,root,root) %{_mandir}/man7/*
+%{_libdir}/libdrop_ambient.so.*
+%{_mandir}/man7/*
```

Comment:
While at it, could we avoid the man page here altogether? We don't really need them. If they don't require too much extra tooling I don't mind some separate subpackage, but should be good just %excluding or maybe moving to -utils.

The explicit attr()s were also strange. On a quick local build test seems like there are no funky permissions. Though I get a lbdrop_ambient.7.gz which is not really compressed.

Reply:
Sounds good, I removed the man page. They only seem to require compressing into .tar.gz files, and in my local builds they are .gz files according to `file`.

I questioned the attr's, so I took them out and checked the permissions in the packages. They looked correct to me, so I committed them. Side-by-side comparison confirms this, aside from python3 package, which I corrected.